File: W4_4_3.java
import java.util.Scanner;
import java.lang.Math;

public class W4_4_3 {
	
	public static void main(String[] args) {
		
		System.out.println("Rayon?");
		Scanner FloatScanner = new Scanner(System.in);
		Float Rayon = FloatScanner.nextFloat();
		
		System.out.println("Aire est: " + Aire(Rayon));		
		System.out.println("Perimetre est: " + Perimetre(Rayon));
	
	}
	
	
	static double Aire(double Rayon) {
	
		return Rayon * Rayon * Math.PI;
	}	


	static double Perimetre(double Rayon) {
		
		return 2 * Rayon * Math.PI;
		
	}


}

File: W4_4_3.py
import math #Wird gebraucht um die Funktion math.pi zu nutzen

rayon = float(input("Rayon:")) #Float, damit auch Dezimalzahlen als Radius eingegeben werden können

def aire():
    return rayon * rayon * math.pi #Exponenten können auch mit '**Exponent', also rayon ** 2 berechnet werden
    
def perimetre():
    return 2 * rayon * math.pi
    
print(aire())
print(perimetre())
